src/coinbase/address/external_address.ts

```diff
@@ -6,9 +6,7 @@ import { Asset } from "../asset";
 import { StakingOperation } from "../staking_operation";
```

Comment:
need to remove stakingRewards from here also ?

Reply:
yeah good call.